src/context/chartLayoutContext.tsx

```diff
@@ -88,6 +89,13 @@ export const ChartLayoutContextProvider = (props: ChartLayoutContextProviderProp
   };
 
   const dispatch = useAppDispatch();
+  /**
+   * This throws an error no matter where I put it (here, useEffect here, separate component)
+   * How is this fixed? Eventually we'll get rud of the chart context provider altogether but...
+   *
+   * Warning: Cannot update a component (`TooltipInternal`) while rendering a different component (`ChartLayoutContextProvider`).
```

Comment:
@PavelVanecek any thoughts about this? I need to set this state, but can't without conflicting with other things being set..

Reply:
We already have `selectIsTooltipActive`, once we have corresponding `selectTooltipCoordinate` then we can delete this dispatch completely.

Until then this can stay as a warning - it's notifying us of discarded re-renders. Not perfect but acceptable for what is effectively a feature branch.

I am not yet sold on if we need `activeLabel` at all.